src/ports/postgres/modules/recursive_partitioning/test/decision_tree.sql_in

```diff
@@ -273,7 +274,7 @@ SELECT tree_train('dt_golf'::text,         -- source table
                          'train_output'::text,    -- output model table
                          'id'::text,              -- id column
                          'temperature::double precision'::text,           -- response
-                         'humidity, windy, "Cont_features"'::text,   -- features
+                         'humidity, windy2, "Cont_features"'::text,   -- features
```

Comment:
we can leave a comment here explaining why we use windy2 with all valid value 'false'